examples/ucp_client_server.c

```diff
@@ -512,7 +551,7 @@ static int parse_cmd(int argc, char *const argv[], char **server_addr,
 
     opterr = 0;
 
-    while ((c = getopt(argc, argv, "a:l:p:c:i:")) != -1) {
+    while ((c = getopt(argc, argv, "a:l:p:c:i:s:m:h")) != -1) {
```

Comment:
need to add new options to `print_usage`

Reply:
it is already there